dpanonymize/__init__.py

```diff
@@ -141,6 +141,9 @@ def lock_lochness(Lochness: 'Lochness',
         pii_table_loc = phoenix_root.parent / 'pii_convert.csv'
         df.to_csv(pii_table_loc)
 
+    # get_file_objects_from_phoenix and get_file_objects_from_module takes
+    # PROTECTED path, but FileInPhoenixBIDS and FileInPhoenix are designed
+    # to set both GENERAL & PROTECTED paths.
```

Comment:
Do you mean `to accept both GENERAL ...` ?

Reply:
No, `FileInPhoenixBIDS` class takes in PROTECTED path, then based on the given PROTECTED path, it also registers GENERAL path attribute.